changes/views.py

```diff
@@ -210,3 +210,34 @@ def search_optouts_message_type():
         'metric_name': total_key,
         'metric_value': total,
     })
+
+
+class ReceiveAdminOptout(mixins.CreateModelMixin,
+                         generics.GenericAPIView):
```

Comment:
We're not using the `CreateModelMixin` here, since we're not using the `create` function, which it provides.

We probably also want the `CreateAPIView` as our base class, since that provides the `post` method, which handles validating the request against the serializer, and then creating the object and returning a 201 if everything is okay, else it returns the relevant error message.